client/errors.go

```diff
@@ -4,9 +4,30 @@ import (
 	"fmt"
 )
 
+var (
+	ErrMissingResponseField MissingResponseFieldErr
```

Comment:
used in private test only, lets make both var and type unexported

Reply:
I did it public for SDK users. For internal tests it's not necessary to have this variables at all

Reply:
if u wanna publish this as an SDK feature, then it must be properly documented: type, var and `Client` docs

Reply:
made unexported